src/docs/asciidoc/reference/configuration/user-management-properties.adoc

```diff
@@ -0,0 +1,245 @@
+:relative-path: ../../
+include::{docdir}/variables.adoc[]
+
+Some properties are available in the `application.json` file to configure the _Standard User Workflow_. We will list all of them in this section:
+
+== Organization
+
+[#configure_organization_name]
+.Organization name
+
+Name of your organization. It is used in the default email sender (`noreply@${organizationName}.com`).
+
+[source, json]
+----
+{
+  "organization": {
+    "name": "ZetaPush"
+  }
+}
+----
+
+.Optional property
+[NOTE]
+====
+This property is optional. If it is not configured, you need to <<reference#configure_email_sender, configure an email sender>>.
+====
+
+== Email
+
+[#configure_email_sender]
+.Email sender
+
+Let you configure the sender of the different emails used in your application. For example for account registration.
+
+.Default email sender
+[NOTE]
+====
+The default email sender is `noreply@${organizationName}.com`.
+So If this property is not set, you need to configure the <<reference#configure_organization_name, organization name>>.
+====
+
+This property support the standard notation for email (name <email@domain.com>):
+
+[source, json]
+----
+{
+  "email": {
+    "sender": "Sender Name <sender@my-domain.com>"  // <1>
+  }
+}
+----
+<1> You can also only type the email (ex: "sender@my-domain.com")
+
+== Account registration
+
+=== Account confirmation
+
+During the account creation workflow of the _StandardUserWorkflow_, an email is sent to ask the end-user to confirm his email. You can configure many properties about this.
+
+.Email subject
+
+You can set the email subject of the confirmation account creation like below:
+
+[source, json]
+----
+{
+  "registration": {
+    "confirmation": {
+      "email": {
+        "subject": "Please confirm your account registration"
+      }
+    }
+  }
+}
+----
+
+.Mandatory property
+[NOTE]
+====
+This property is mandatory.
```

Comment:
For now, this is a mandatory property. In a future version, the subject could be extracted from the email template. This allows you to define a single source of information about the sent email and moreover, it simplifies internationalization of your email.